packages/hub/src/utils/sha-node.ts

```diff
@@ -8,7 +8,26 @@ export async function* sha256Node(
 		abortSignal?: AbortSignal;
 	}
 ): AsyncGenerator<number, string> {
-	const sha256Stream = createHash("sha256");
+	return yield* shaNode("sha256", buffer, opts);
+}
+
+export async function* sha1Node(
+	buffer: ArrayBuffer | Blob,
+	opts?: {
+		abortSignal?: AbortSignal;
+	}
+): AsyncGenerator<number, string> {
+	return yield* shaNode("sha1", buffer, opts);
+}
```

Comment:
Computing the git hash is slightly different than a sha1 on the content. That's why I called the helper [`git_hash`](https://github.com/huggingface/huggingface_hub/blob/1f9dbf896e791bc7beb7a3a855856e43d08ee3cf/src/huggingface_hub/utils/sha.py#L32) in Python. 

The logic looks like this:
```py
def git_hash(data: bytes) -> str:    
    sha = sha1()
    sha.update(b"blob ")
    sha.update(str(len(data)).encode())
    sha.update(b"\0")
    sha.update(data)
    return sha.hexdigest()
```

Reply:
sh*t yes thanks I forgot about that, I was too focused on understanding the codebase 😂

Reply:
01c49b95b00b6e1e2ad8a6a98c20294cb53c455e